asana/README.md

```diff
@@ -7,10 +7,98 @@ A connector to integrate Asana, the project management tool, to Cohere. Featurin
 The Asana connector only offers full-text search for Task objects.
 
 ## Configuration
+The Asana connector provides two authentication methods: [Personal Access Token](https://developers.asana.com/docs/personal-access-token) and [OAuth 2.0](https://developers.asana.com/docs/oauth)
 
-To use this connector, head to your Asana Developer Console. From here, you can create a new Personal Access Token. Use this value for the `ASANA_ACCESS_TOKEN` environment variable. Then, head to your Admin Console. The URL at the top of your web browser should look like `https://app.asana.com/admin/<workspace_gid>`, grab the `<workspace_gid>` value and use it for `ASANA_WORKSPACE_GID`.
+If you are using a personal access token, you need to set the following environment variables:
```

Comment:
To clarify the authentication methods, could you add sub sections please, eg a 
```
### (Method 1) Personal Access Token

### (Method 2) OAuth 2.0
```

Reply:
It's done. Thank you.